analyzers/its/SonarAnalyzer.Testing.ImportBefore.targets

```diff
@@ -55,8 +55,10 @@
 
     <PropertyGroup Condition="$(SonarAnalyzerRegressionTestRunning)">
       <!-- Set the path to the SARIF report -->
-      <ErrorLog>$(SonarProjectOutputPath)-$(ProjectGuid).json</ErrorLog>
-      <RazorCompilationErrorLog>$(SonarProjectOutputPath)-$(ProjectGuid).Views.json</RazorCompilationErrorLog>
+      <ErrorLog Condition="'$(TargetFramework)' != ''">$(SonarProjectOutputPath)-$(ProjectGuid)-$(TargetFramework).json</ErrorLog>
```

Comment:
there seems to be an alignment mismatch between the various xml tags (or mix of tab and space?)

Reply:
Indeed, I should not blindly trust Notepad++ :)